eng/vmr-sync.sh

```diff
@@ -20,26 +20,26 @@
 ###   Synchronize the VMR to a specific commit of dotnet/runtime using custom fork:
 ###     ./vmr-sync.sh \
 ###        --repository runtime:e7e71da303af8dc97df99b098f21f526398c3943 \
-###        --remote runtime:https://github.com/yourfork/runtime \
-###        --tmp-dir "$HOME/repos/tmp"\
+###        --remote runtime:https://github.com/yourfork/runtime          \
+###        --tmp-dir "$HOME/repos/tmp"
 ###
 ### Options:
 ###   -t, --tmp, --tmp-dir PATH
 ###       Required. Path to the temporary folder where repositories will be cloned
-###   --repository name:GIT_REF
+###   -r name:GIT_REF, --repository name:GIT_REF
 ###       Optional. Repository + git ref separated by colon to synchronize to.
 ###       This can be a specific commit, branch, tag..
 ###       When omitted, the script will synchronize the installer commit based on the version of the parent
 ###       where this script is stored.
+###   --remote name:URI
```

Comment:
Yeah, there's some logic behind it but it's my personal impression of how these clump into logic groups as they influence similar things, e.g. (vmr, tmp, branch), (repository, remote, recursive), (tpn template, readme template), (debug, help).

Do you think just [required], [optional] and both alphabetically conform better to the expectations?

Reply:
The clumping only makes sense if it is called out to help people not intimately familiar with the script.  I am not sure what to name these groups.  Otherwise they should be grouped by required/optional and then sorted alphabetically IMO.

Reply:
Makes sense. I sorted them alphabetically.